### PR TITLE
[v1.5rc] RemoveTag feature

### PR DESCRIPTION
**Pull request by yewshengkai**
- [x] Implement undoable RemoveTag feature
- [ ] Implement TestCase for RemoveTag feature
I'll leave TestCase to owner of feature.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/RemoveTag.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+import seedu.address.model.tag.Tag;
+
+//@@author yewshengkai
+/**
+ * Remove tag of all person in the address book.
```
Comment (karrui) [inline src/main/java/seedu/address/logic/commands/RemoveTag.java]: We should change this to `Remove specified tag from all contacts in addressbook`

Comment (yewshengkai) [reply src/main/java/seedu/address/logic/commands/RemoveTag.java]: oh you mean this line. I thought you referred to message usage. Thanks

File: src/main/java/seedu/address/model/ModelManager.java [hunk 2]
```diff
@@ -82,6 +86,20 @@ public void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
     }
 
     @Override
+    public void removeTag(Tag tag) throws PersonNotFoundException, DuplicatePersonException {
+        for (int i = 0; i < addressBook.getPersonList().size(); i++) {
+            ReadOnlyPerson oldPerson = addressBook.getPersonList().get(i);
+
+            Person newPerson = new Person(oldPerson);
```
Comment (karrui) [inline src/main/java/seedu/address/model/ModelManager.java]: Here

Comment (yewshengkai) [reply src/main/java/seedu/address/model/ModelManager.java]: Oki x)

Comment (karrui) [reply src/main/java/seedu/address/model/ModelManager.java]: Remove this line since newPerson is being instantiated below
Change to
`Person newPerson;`

Comment (yewshengkai) [reply src/main/java/seedu/address/model/ModelManager.java]: Also changed this line
newPerson = new Person(oldPerson, newPerson.getHomepage());
to
newPerson = new Person(oldPerson, oldPerson.getHomepage());

File: src/main/java/seedu/address/logic/commands/RemoveTag.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+//@@author yewshengkai
+/**
+ * Tag specified will be remove from addressbook.
+ */
+public class RemoveTag extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "removetag";
+    public static final String COMMAND_ALIAS = "rt";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Tag specified will be remove from addressbook. "
```
Comment (karrui) [inline src/main/java/seedu/address/logic/commands/RemoveTag.java]: Typo -- `remove` -> `removed`

Comment (yewshengkai) [reply src/main/java/seedu/address/logic/commands/RemoveTag.java]: Noted. d(>w<)b

**Pull request merged**